test/config.ts

```diff
@@ -1 +1,2 @@
-export const defaultEndpoint = "ws://0.0.0.0:9944";
+export const defaultParachainEndpoint = "ws://0.0.0.0:9944";
+export const defaultFaucetEndpoint = "http://0.0.0.0:3034";
```

Comment:
Do we also want to add the default endpoints for the vault and staked-relayer?

Reply:
These are the only two endpoints that are being used for integration testing at the moment. I can add the others now, or when I actually write the tests for the other clients